web_monitoring/internetarchive.py

```diff
@@ -208,17 +211,15 @@ def list_versions(url, *, from_date=None, to_date=None, skip_repeats=True):
     if to_date:
         params['to'] = to_date.strftime(URL_DATE_FORMAT)
 
-    has_versions = False
     last_hashes = {}
     for version in search_cdx(params):
         # TODO: may want to follow redirects and resolve them in the future
         if not skip_repeats or last_hashes.get(version.url) != version.digest:
-            has_versions = True
             last_hashes[version.url] = version.digest
             # TODO: yield the whole version
             yield version
 
-    if not has_versions:
+    if not last_hashes:
```

Comment:
Huh, I was about to say this would never pass, but now I see empty dicts are falsy. How did I never know that was the case in Python??? 👍